core/core-impl/src/test/java/ai/timefold/solver/core/api/solver/SolverFactoryTest.java

```diff
@@ -128,10 +137,24 @@ void createFromInvalidXmlFile_failsShowingBothPathAndReason() throws IOException
 
     @Test
     void create() {
-        SolverConfig solverConfig = PlannerTestUtils.buildSolverConfig(TestdataSolution.class, TestdataEntity.class);
-        SolverFactory<TestdataSolution> solverFactory = SolverFactory.create(solverConfig);
-        Solver<TestdataSolution> solver = solverFactory.buildSolver();
-        assertThat(solver).isNotNull();
+        { // No config override
+            SolverConfig solverConfig = PlannerTestUtils.buildSolverConfig(TestdataSolution.class, TestdataEntity.class);
+            SolverFactory<TestdataSolution> solverFactory = SolverFactory.create(solverConfig);
+            Solver<TestdataSolution> solver = solverFactory.buildSolver();
+            assertThat(solver).isNotNull();
+        }
```

Comment:
Although these explicit code blocks don't hurt, we do not use that patterns.
It's an extra code which IMO makes no difference.